lightning/src/util/time.rs

```diff
@@ -0,0 +1,172 @@
+// This file is licensed under the Apache License, Version 2.0 <LICENSE-APACHE
+// or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// You may not use this file except in accordance with one or both of these
+// licenses.
+
+//! A very simple serialization framework which is used to serialize/deserialize messages as well
+//! as ChannelsManagers and ChannelMonitors.
+
+use core::ops::Sub;
+use core::time::Duration;
+use core::cell::Cell;
+/// A measurement of time.
+pub trait Time: Copy + Sub<Duration, Output = Self> where Self: Sized {
+	/// Returns an instance corresponding to the current moment.
+	fn now() -> Self;
+	
```

Comment:
nit: EOL whitespace here (and a number of similar places in this file).

Reply:
Sorry. I will fix this after other issues resolved(I need to find a way to config my editor to avoid these)